docs/UserGuide.md

```diff
@@ -307,9 +308,10 @@ Format: `remind`
 * Archived interviews will not be displayed even if they fall within the next week.
 * If there are no upcoming interviews, a blank window will be displayed.
 * `remind` can only be used on the main application list page (i.e. not to be used on list-archive page).
+* To refresh the list of upcoming interviews, close the reminder window (if it is currently opened) and input `remind` command again.
 
-### Clearing all entries: `clear`
-Clears all entries from CinternS.
+### Clearing all entries: `clear` 
+Clears all entries from CinternS. 
```

Comment:
Is this intended? I'm not sure what's this is for

Reply:
oops yea this is not intended 😅